.eslintrc.cjs

```diff
@@ -7,11 +7,21 @@ module.exports = {
   },
   /** Default rules for Javascript, Typescript & Tests */
   rules: {
+    // Use const/let
     'no-var': 'error',
+
+    /**
+     *  Force `===` but allow `== null`
+     *  to prevent `x === undefined || x === null`
+     */
     eqeqeq: ['error', 'always', { null: 'ignore' }],
+
+    // https://eslint.org/blog/2022/07/interesting-bugs-caught-by-no-constant-binary-expression/
+    'no-constant-binary-expression': 'error',
```

Comment:
Actually, might want to add a TODO here to remove this when we upgrade to eslint 9.

Reply:
v9.0.0 changes the entire configuration structure so this would need a complete rework anyway.

Reply:
Oh right. nvm :+1: